test/unit/algorithm/CMakeLists.txt

```diff
@@ -67,10 +67,10 @@ macro(RAJA_GENERATE_ALGORITHM_UTIL_SORT_TESTS SORT_BACKEND_in SORT_SIZE_in UTIL_
     if(${SORT_BACKEND} STREQUAL "Hip" AND (${UTIL_SORT} STREQUAL "Heap" OR 
 	                                   ${UTIL_SORT} STREQUAL "Insertion" OR
 					   ${UTIL_SORT} STREQUAL "Intro"))
-        raja_add_test( NAME test-algorithm-util-sort-Known-Hip-Failure-${UTIL_SORT}-${SORT_BACKEND}
+        raja_add_test( NAME test-algorithm-util-sort-${UTIL_SORT}-${SORT_BACKEND}
```

Comment:
we should be able to remove the conditional here

Reply:
Sorry. I missed that.

Reply:
change pushed.